src/ui/components/VPNControllerView.qml

```diff
@@ -618,16 +657,28 @@ Item {
             height: 16
         }
 
-        VPNInterLabel {
-            id: logoSubtitle
-            objectName: "controllerSubTitle"
-
-            lineHeight: Theme.controllerInterLineHeight
-            Accessible.ignored: true
-            width: parent.width - Theme.windowMargin
```

Comment:
This is tricky! Unfortunately if we remove width here the `logoSubtitle` text loses its ability to wrap, which becomes problematic when viewing the app in more verbose locales (try  going to Settings -> Languages and changing the default language to German and you'll see what I mean). But if we leave it, the timer isn't visible or is way out of alignment.

I think the easiest thing to do here may actually be to treat "Secure and private • 00 : 00 : 00" string as a special case and handle it separately from the other `logoSubtitle` strings - sort of like we do for `VPNConnectionStability { }`. In that case we'd just need to:
- revert the changes `logoSubtitle` and hide it when `VPNController.StateOn`
- add a new `RowLayout { } ` that is only visible when `VPNController.StateOn` to handle the "Secure and private •" string and `VPNSemiMonoLabel { }`. Something along the lines of:
```
RowLayout {
  id: someNewId
  opacity: .8
  anchors.horizontalCenter: parent.horizontalCenter
  visible: false

  VPNInterLabel {
    objectName: "secureAndPrivateSubtitle"

    lineHeight: Theme.controllerInterLineHeight
    Accessible.ignored: true
    onPaintedHeightChanged: if (visible) col.handleMultilineText()
    onVisibleChanged: if (visible) col.handleMultilineText()
    color: Theme.white

    //% "Secure and private"
    //: This refers to the user’s internet connection.
    text: qsTrId("vpn.controller.active") + " • "
  }

  VPNSemiMonoLabel {
    id: connectionTime
    Accessible.ignored: true
  }
}